bip-schnorr.mediawiki

```diff
@@ -147,16 +158,18 @@ All provided signatures are valid with overwhelming probability if and only if t
 ==== Signing ====
 
 Input:
-* The secret key ''d'': an integer in the range ''1..n-1''.
+* The secret key ''d' '': an integer in the range ''1..n-1''
 * The message ''m'': a 32-byte array
 
-To sign ''m'' for public key ''dG'':
+To sign ''m'' for public key ''bytes(dG)'':
+* Let ''P = dG''
```

Comment:
You don't know `d` yet, should be `bytes(d'G)` and `P=d'G`. I think it might be clearer to call the input `sk` (like `pk`) rather than `d'` though?